src/groupByKey.ts

```diff
@@ -1,7 +1,22 @@
 interface Item {
-  [key: string]: any;
+  [key: string]: any,
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+interface GroupedByKey {
+  [key: string] : Item[],
+}
+
+export function groupByKey(items: Item[], field: keyof Item)
+  : GroupedByKey {
```

Comment:
Let's use correct code style:
```suggestion
export function groupByKey(
  items: Item[],
  field: keyof Item,
): GroupedByKey {
```

Reply:
TypeScript styling is kind of hard for newbie with all that additional syntax) 😄